python-avd/pyavd/_eos_designs/structured_config/base/__init__.py

```diff
@@ -640,7 +640,33 @@ def ip_radius_source_interfaces(self) -> None:
         if source_interfaces := self._build_source_interfaces(
             inputs.mgmt_interface, inputs.inband_mgmt_interface, "IP Radius", output_type=EosCliConfigGen.IpRadiusSourceInterfaces
         ):
-            self.structured_config.ip_radius_source_interfaces = source_interfaces
+            self.structured_config.ip_radius_source_interfaces.extend(source_interfaces)
+
+    @structured_config_contributor
+    def radius_servers(self) -> None:
+        """Parse AAA radius server configurations and update structured config with server and source interface details."""
+        if not self.inputs.aaa_settings.radius:
+            return
+
+        vrfs = self.inputs.aaa_settings.radius.vrfs
```

Comment:
do you need this? you can use it directly line 655
```suggestion
        vrfs = self.inputs.aaa_settings.radius.vrfs
```

Reply:
updated